rxlifecycle-kotlin/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile kotlinStdlib
+    compile project(':rxlifecycle')
+
+    compile rootProject.ext.rxJava
+
+    compile rootProject.ext.appCompat
+
+    testCompile rootProject.ext.junit
+    testCompile rootProject.ext.mockito
+    testCompile rootProject.ext.robolectric
```

Comment:
Does this part of the project really need all these dependencies? I'm guessing it only needs the top two.